lib/ansible/modules/cloud/amazon/route53_health_check.py

```diff
@@ -314,9 +314,9 @@ def main():
     # Default port
     if port_in is None:
         if type_in in ['HTTP', 'HTTP_STR_MATCH']:
-            port_in = 80
+            port_in = '80'
```

Comment:
It works either way, but to conform more to what _should_ be in place, given those boto docs, I have submitted a change to make sure the comparison object uses an integer.